xla/service/gpu/priority_fusion.cc

```diff
@@ -164,6 +164,11 @@ class GpuPriorityFusionQueue {
       instructions.push_back(instruction);
     }
 
+    for (auto instruction : computation->MakeInstructionPostOrder()) {
+      if (instruction->IsFusible()) {
```

Comment:
`IsFusible` is also checked inside `UpdatePerformanceModelCache`. It's enough to do this check only once. Either here or inside the function.